repos/system_upgrade/el7toel8/libraries/dnfplugin.py

```diff
@@ -106,6 +106,8 @@ def _transaction(context, stage, target_repoids, tasks, test=False):
         ]
         if config.is_verbose():
             cmd.append('-v')
+        if cmd_prefix:
+            cmd[0:0] = cmd_prefix
```

Comment:
while this works, I think it's easier digestable as
```suggestion
            cmd = cmd_prefix + cmd
```

Reply:
done

Reply:
anyway, it's nice trick to know :)